governance/tag-governance.md

```diff
@@ -0,0 +1,541 @@
+- MUST be nominated by the TAG’s Tech Leads.  
+- Membership MUST be tracked alongside the TAG metadata.  
+- SHOULD have prior experience within the CNCF (e.g. as an active TAG community member), but MAY be
+  approved by their parent governing body for an explicitly documented reason such as extensive
+  domain expertise, formation of a new governing body (e.g. new TAG or Working Group), or
+  demonstrating significant aptitude for people & community management. 
+
+##### Duties
+
+- SHOULD set direction and make or approve design decisions for their subproject - either directly
+  or through delegation.   
+  - Design decisions that make changes outside of the subproject’s defined scope MUST be approved
+    by the Tech Lead(s) of the TAG.  
+- MUST actively participate in triaging issues and reviewing PRs related to their subproject.  
+- MUST regularly report subproject status to TAG leadership.  
+- SHOULD be an escalation point for technical discussions and decisions in the subproject.  
+- SHOULD set milestone priorities.   
+- SHOULD ensure a healthy process for discussion and decision making is in place. MAY make decisions
+  to resolve conflicts.
+
+##### Activity Expectations
+
+- Subproject Leads serve 1 year terms, and  MAY submit for re-election.  
+- Subproject Leads MAY decide to step down at any time and propose a replacement to complete their
+  term.   
+  - The candidate SHOULD be supported by a majority of the current active leads, with support backed
+    up by documentation.  
+  - The candidate is REQUIRED to be approved by the parent TAG’s Tech Leads.  
+- Subproject Leads taking an extended leave of 1 or more months SHOULD coordinate with other leads
+  to ensure the role is adequately covered during their leave.  
+- Subproject Leads going on leave for 1-3 months MAY work with other Leads to identify a temporary
+  replacement.  
+  - Active Leads are REQUIRED to notify their parent governing body (e.g., TAG notifying their TOC
+    liaison) and receive confirmation of notice before pursuing removal of an inactive or Lead found
+    to be in dereliction of duty.  
+  - Inactive Leads MAY be removed through a super-majority vote of the parent governing body if
+    there is no other explicitly defined removal policy.  
+  - If there is no defined removal policy and not enough active parent governing body leads to hold
+    a vote, the active members SHOULD escalate to the next-level parent governing body, where those
+    members may count as additional voters.
+
+##### Escalations
+
+- Lead membership disagreements MAY be escalated to their parent governing body, the CoCC as
+  applicable, or in the event of further dispute, up to the top governing body itself (TOC).
+
+#### Other Roles
+
+Subprojects MAY have other roles, but are required to be documented in the subproject charter and
+adhere to the requirements outlined in the Other Roles section of the CNCF Community Group
```

Comment:
I couldn't find "Other Roles" under the CNCF Community Group section in this document. A link to what we are referring to would be helpful. 

Also there is no leadership or roles defined for community groups, will they only have organizer role as mentioned [here](https://github.com/cncf/communitygroups/blob/main/organizer_roles.md)?

Reply:
That is correct, they will only have the organizer role.